mp/src/game/client/hud_crosshair.h

```diff
@@ -11,6 +11,7 @@
 #pragma once
 #endif
 
+#include "weapon/weapon_base.h"
 #include "hudelement.h"
 #include <vgui_controls/Panel.h>
 
```

Comment:
Instead of including weapon base here, include it in the cpp file and do 
`class CWeaponBase;`
below the last include in this file.

This is forward declaring and prevents other classes accidentally getting more included than they should.